kitsune/sumo/email_utils.py

```diff
@@ -22,11 +22,13 @@ def send_messages(messages):
     if not messages:
         return
 
-    conn = mail.get_connection(fail_silently=True)
-    conn.open()
-
-    for msg in messages:
-        conn.send_messages([msg])
+    try:
+        conn = mail.get_connection()
+        conn.open()
+        for msg in messages:
+            conn.send_messages([msg])
+    finally:
+        conn.close()
```

Comment:
I've always been told that I shouldn't rely on implicit destruction in Python. I looked into this a bit more and it would probably be fine in this case. I can switch it back.

I definitely agree we should add re-send logic at some point in the stack. I wanted to see what happened with this before we made too many decisions though.